src/views/Home/components/FAQ/index.view.tsx

```diff
@@ -1,8 +1,100 @@
 import React from "react";
```

Comment:
#71 allows us to get rid of importing react if not needed; however, since we do use it, let's use this react import: 
`import * as React from "react"`. Future versions of react is deprecating the usage of this current import I believe.